webapp/packages/core-ui/src/Tabs/Tab/TabTitle.tsx

```diff
@@ -5,14 +5,15 @@
  * Licensed under the Apache License, Version 2.0.
  * you may not use this file except in compliance with the License.
  */
-import styled from 'reshadow';
+import { s, useS } from '@cloudbeaver/core-blocks';
 
-import { BASE_TAB_STYLES } from './BASE_TAB_STYLES';
+import { baseTabStyles } from '../..';
```

Comment:
wrong import
(you probably can delete all `placeholders` in tabs because we don't use them

Reply:
well we have situations like this where it can be showed

![Снимок экрана 2024-03-11 в 11 46 41](https://github.com/dbeaver/cloudbeaver/assets/26230285/c10f8863-da73-4124-8e36-26ae03a1523a)